Dockerfile

```diff
@@ -107,7 +169,12 @@ RUN . $CONDA_PATH/etc/profile.d/conda.sh && \
             pip install $_wheel --constraint ./dist/constraints.txt; \
         done \
     fi && \
-    pip install $(ls "./dist/mlserver-"*.whl) --constraint ./dist/constraints.txt && \
+    if [[ ${#_extras[@]} -gt 0 ]]; then \
+        extras_list=$(IFS=, ; echo "${_extras[@]}") && \ 
```

Comment:
Following the same reasoning as the other comment, unless we have some reason to avoid it I'd install all the extras in the Dockerfile - that way the `seldonio/mlserver` image is always ready as-is to enable tracepoints.

Reply:
Happy to do that if the decision (given above comments) is to install all extras. I don't know if pip has a simple way of doing that here, I might still have to build a list.

Reply:
We can also see if Poetry supports an `all` extras target? Otherwise we can always add one.

Reply:
(and / or just hardcode a `pip install mlserver[extras1, extras2, etc.]`)

Reply:
Poetry currently supports --all-extras on install but not export (this is tracked in python-poetry/poetry-plugin-export#45). I've added an `everything` extra. I've intentionally avoided the name `all` as that might be reserved in the future by poetry/pip, but we can move to it once there is enough support.

Reply:
TBH I would just use `all` as it's more standard. If poetry decides to always add one automatically we can just remove it and use Poetry's built-in target.